handle/middlewares/cors_test.go

```diff
@@ -0,0 +1,58 @@
+package middlewares
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+)
+
+var GetCorsHeadersRequest = map[string]string {
+	"Origin":                        "http://foobar.com",
+	"Access-Control-Request-Method": "GET",
+	"Access-Control-Request-Headers": "Accept, Authorization, Content-Type",
+	"Access-Control-Allow-Methods": "*",
+}
+
+var GetCorsHeadersResponse = map[string]string {
+	"Vary":                        "Origin",
+	"Access-Control-Allow-Origin": "http://foobar.com",
+	"Access-Control-Request-Headers": "Accept, Authorization, Content-Type",
+	"Access-Control-Allow-Credentials": "true",
+}
+
+var allHeaders = []string{
+	"Vary",
+	"Access-Control-Allow-Origin",
+	"Access-Control-Allow-Methods",
+	"Access-Control-Allow-Headers",
+	"Access-Control-Allow-Credentials",
+	"Access-Control-Max-Age",
+}
+
+func assertHeaders(t *testing.T, resHeaders http.Header, expHeaders map[string]string) {
```

Comment:
Maybe it is not necessary, this assertion will be used only in this file. What do you think?